### DISCUSSION TITLE
example with Async WebSocket Plugin

### DISCUSSION THREAD

**Discussion by oleometal** [khoih-prog/Portenta_H7_AsyncWebServer#13] [General]
Hello, I'm using Portenta H7 AsyncWebServer and it seems great, and I wanted to ask you if you could make an example with Async WebSocket Plugin please

**Comment by khoih-prog** [comment]
HI @oleometal 

Please have a look now at [**Async_WebSocketsServer**](https://github.com/khoih-prog/AsyncWebServer_RP2040W/tree/main/examples/Async_WebSocketsServer) example in `twin` [**AsyncWebServer_RP2040W**](https://github.com/khoih-prog/AsyncWebServer_RP2040W) library and you can adapt it easily to Portenta_H7, if you need it urgently.

I'll do it later if having time.

Good Luck,
